include/h2o/socket/evloop.h

```diff
@@ -47,6 +47,7 @@ typedef struct st_h2o_evloop_t {
     struct timeval _tv_at;
     h2o_timerwheel_t *_timeouts;
     h2o_sliding_counter_t exec_time_counter;
+    h2o_sliding_counter_t exec_time_nanosec_counter;
```

Comment:
Is there a reason to _add_ `exec_time_nanosec_counter` as a new member, instead of _replacing_ `exec_time_counter` with the nanosec counter?

While I think it is a good idea to retain `h2o_evloop_get_execution_time` and `h2o_evloop_get_execution_time_nanosec`, I am not sure if there is a reason to retain two internal members.

Reply:
The reason is that I wasn't sure if removing the existing counter was an appropriate thing to do or not. Objectively, I agree that there's no _strong_ reason to keep both. With your blessing, I'll commit a change that replaces `exec_time_counter` with `exec_time_nanosec_counter`.

Reply:
Done: 9f01afea

- Replaced  `exec_time_counter` with `exec_time_nanosec_counter`
- Renamed `h2o_evloop_get_execution_time` to `h2o_evloop_get_execution_time_millisec` for clarity (like we did with the `now_millisec`)